osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -123,44 +161,61 @@ private double computeAimValue()
             // It is important to also consider accuracy difficulty when doing that
             aimValue *= 0.98 + Math.Pow(Attributes.OverallDifficulty, 2) / 2500;
 
+            // Scale by % FC'd to encourage FC.
+            aimValue *= 0.8 + 0.2 * (Score.MaxCombo / (double)Attributes.MaxCombo);
+
+            if (categoryRatings != null)
+            {
+                categoryRatings.Add("Aim", aimValue);
+                categoryRatings.Add("Aim Combo Stars", aimComboStarRating);
+                categoryRatings.Add("Aim Miss Count Stars", aimMissCountStarRating);
```

Comment:
These are confusing. I'd probably rename to something like `Aim Estimate (combo-based)`, `Aim Estimate (miss count-based)`, `Aim (final)`.

Reply:
Would something like Achieved Aim Difficulty (combo-based) make sense?

Reply:
Sure, that works.